modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -15,6 +15,17 @@ If you reuse the `BareMetalHost` object definition from an existing control plan
 Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
 ====
 
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
```

Comment:
This sounds like Authorization, can you reword that openshift API access is working fine?

Reply:
Hi @pmannidi - I checked in our documentation - and with some other writers on my team - and this is the standard way we write this across our documentation. I think we should keep this consistent and leave it, hope that's ok.